src/vs/code/electron-main/app.ts

```diff
@@ -826,34 +826,37 @@ export class CodeApplication extends Disposable {
 			//   To: vscode-remote://wsl+ubuntu/mnt/c/GitDevelopment/monaco
 
 			const secondSlash = uri.path.indexOf(posix.sep, 1 /* skip over the leading slash */);
+			let authority, path;
```

Comment:
I dont want to be nit picking, but our code style is:

```ts
let authority: string;
let path: string;
```

Reply:
I wasn't aware of that, but yes, sure.